### PR TITLE
Removed deprecated .png file

### LINKED ISSUE #9237: Remove usages of missing icon_cancel.png
### Problem

Edit page tries to load a `icon_cancel.png` that doesn't exist.
I don't know when we deleted it, probably long ago, but it doesn't do anything.
So lets remove references to it.

#### Evidence / Screenshot
image
image

#### Relevant URL(s)
- https://openlibrary.org/authors/OL5046634A/John_Green/edit

### Reproducing the bug

_No response_

### Context

_No response_

### PR DESCRIPTION
**Pull request by Spedi**
Closes #9237 

Removed usages of `icon_cancel.png`, since it doesn't exist and it was still referenced in some part of the codebase.

### Stakeholders
@RayBB

### CODE REVIEW

**Comment by rebecca-shoptaw** [comment]
Hello! Quick heads-up re: the `pre-commit` `Generate POT` check that this PR failed -- this is just because we added a new automatic action to update the translation files.

As soon as you update your master branch and rebase this one, the translations file will now be up to date and the test will pass, and you won't encounter this issue again with any branches off the updated master. 🙂 

Steps to follow if needed:
```
git switch master
git pull upstream master
git push origin master
git switch your-branch
git rebase master
```

**Comment by Spedi** [comment]
Thank you so much @rebecca-shoptaw!
Done those steps in Git, do I need to do anything else to make the PR work correctly?

**Comment by rebecca-shoptaw** [comment]
@Spedi Quick update! It looks like the remote version your `master` branch is still out of sync with the OL master -- this seems to be the result of a [manual merge conflict handling](https://github.com/internetarchive/openlibrary/compare/master...Spedi:openlibrary:master). To get everything back up to date, I'd recommend running:
```
# switches to master branch
git switch master

# undoes the last few commits on it, including the merge conflict
git reset --hard HEAD~3

# pulls in the OL master
git pull upstream master

# force pushes your changes to your remote branch
git push -f origin master
```
Once this is done, you should be able to switch to your branch and get it up to date as described:
```
git switch your-branch
git rebase master
git push origin HEAD
```

Or `git push -f origin HEAD` if that fails!

P.S. Because your PR does involve adding new translated text to the site (the alt and title for the icon), the pre-commit hook should successfully update the translation file for you when the PR is submitted.

**Comment by Spedi** [comment]
@rebecca-shoptaw done the steps, does everything look right now?
@RayBB implemented the suggested changes too.

**Comment by rebecca-shoptaw** [comment]
@Spedi thanks for running those commands! Weirdly it looks like some of them must have failed, as your [remote master branch](https://github.com/Spedi/openlibrary) is still showing as out of date:
Screenshot 2024-05-10 at 3 37 57 PM

Happy to go back-and-forth on Slack or here to get this figured out! A question to get us started:

When you're on the local master branch (in VSCode or whatever IDE you use) and you navigate to the `messages.pot` file (`/openlibrary/i18n/messages.pot`), do you see the following creation date?
```
"POT-Creation-Date: 2024-05-01 18:58-0400\n"
```
If so, your local master is up to date, and it will just be a matter of getting your remote master and local/remote branch for the PR to match.

If not, I can help you get your local master back up to date and work from there. 🙂

**Comment by Spedi** [comment]
@rebecca-shoptaw I see `"POT-Creation-Date: 2024-04-17 18:20+0000\n"`, so I suppose that the local master is not up to date, what are the steps I need to do in order to resolve this issue?

**Comment by rebecca-shoptaw** [comment]
@Spedi All good! The following should fix it:
```
# on master branch
git reset --hard HEAD~10
git pull upstream master
```
You can check the date on the `messages.pot` file once this is done and it should be updated!

**Comment by Spedi** [comment]
@rebecca-shoptaw I done those steps on git, but it doesn't seem like it's working, the `messages.pot` file still shows the old date. What do you think this happen?
Thank you for your patience ;)

**Comment by scottbarnes** [comment]
pre-commit.ci run

**Comment by scottbarnes** [comment]
@Spedi, I can't tell if you can fix this by simply running `pre-commit run generate-pot --all-files` locally and adding the updated `/code/openlibrary/i18n/messages.pot` to your PR, or whether this won't work until #9258 is merged and you rebase off master (or you make similar changes).

We may need to change the pre-commit rule a bit if #9258 doesn't stop this from happening.

**Comment by rebecca-shoptaw** [comment]
@Spedi Very weird that the upstream pull isn't going through! I'll start a back and forth with you in Slack so you can send screenshots/etc. and we can figure it out. 🙂 

And I can confirm that the error messages @scottbarnes noticed, while annoying and on their way to being fixed, shouldn't be what's holding this up! Once the branch is correctly up to date you should be all good.

**Comment by cdrini** [comment]
It looks like something got tangled with your master branch @Spedi ; this can happen when one accidentally commits something to the local master branch. Run this to clean it up to avoid issues in future PRs:

```sh
git checkout master
git fetch upstream master
git reset --hard FETCH_HEAD
```

This resets it to be identical to the internetarchive/openlibrary master branch.

I've fixed up this branch now since it's a small branch which we should be able to quickly merge :+1:

File: static/css/components/buttonsAndLinks.less
Comment (cdrini) [inline static/css/components/buttonsAndLinks.less:6]: This padding is likely for the icon so can be removed.

```suggestion
```

**Review by RayBB — APPROVED** [review]
Just confirmed on testing and it looks good to me!
image

**Pull request merged**
